builtin/providers/aws/resource_aws_db_event_subscription.go

```diff
@@ -0,0 +1,326 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/rds"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsDbEventSubscription() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsDbEventSubscriptionCreate,
+		Read:   resourceAwsDbEventSubscriptionRead,
+		Update: resourceAwsDbEventSubscriptionUpdate,
+		Delete: resourceAwsDbEventSubscriptionDelete,
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type:         schema.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: validateDbEventSubscriptionName,
+			},
+			"sns_topic": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"event_categories": &schema.Schema{
+				Type:     schema.TypeSet,
+				Optional: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+				Set:      schema.HashString,
+			},
+			"source_ids": &schema.Schema{
+				Type:     schema.TypeSet,
+				Optional: true,
+				ForceNew: true,
```

Comment:
Just FYI it looks like we might be able to make this update-able in the future via http://docs.aws.amazon.com/cli/latest/reference/rds/add-source-identifier-to-subscription.html (or rather the underlying API that command uses). It's absolutely 👌 to keep it `ForceNew` for the initial implementation though. Just FYI.